learn/experimental/autobatching.md

```diff
@@ -0,0 +1,42 @@
+# Auto-batching
+
+If enabled, multiple document addition requests will be automatically combined into a batch and processed together to speed up indexation.
+
+We would like your feedback on this feature! [Join the discussion](https://github.com/meilisearch/meilisearch/discussions/2070).
+
+## Enable auto-batching
+
+To enable auto-batching, start Meilisearch while supplying the `--enable-autobatching` CLI flag:
+
+```
+./meilisearch --enable-autobatching
+```
+
+For document addition requests to be added to the same batch, they need to:
+
+- Target the same index
+- Have the same update method (e.g. [POST](/reference/api/documents.md#add-or-replace-documents) or [PUT](/reference/api/documents.md#add-or-update-documents))
+- Be immediately consecutive
+
+After enabling autobatching, the field `batchUid` will appear in all [Task API](/reference/api/tasks.md) responses.
+
+::: warning
+
+If even a single task in a batch fails, the entire batch will fail.
+
+:::
+
+## Customization options
+
+There are three command-line options allowing you to customize auto-batching behavior:
```

Comment:
Are these also present as env vars?

Reply:
No env var specified for now! Saying that notification late. Sorry guys!